src/button/component/button.jsx

```diff
@@ -56,18 +53,15 @@ export default class OpenStadComponentButton extends OpenStadComponent {
       labelHTML = <div className="osc-button-label"><div className="osc-elipsis">{self.config.label}</div></div>
     }
 
+    let onClick = typeof self.props.onClick != 'undefined' ? self.props.onClick : self.state.onClick || null;
```

Comment:
`onClick` doesn't seem to exist in the state, so I think we can remove it here?

```suggestion
    let onClick = typeof self.props.onClick != 'undefined' ? self.props.onClick : null;
```